articles/introduction-to-neural-networks/index.md

```diff
@@ -0,0 +1,189 @@
+### Introduction
+
+In this article, we will be talking about neural networks. A functional technique of deep learning whose motivation is by how the brain works.
+
+This article aims to make you understand what neural networks are. It will also give you a depth interpretation of how neural networks operates. This article will explain distinct kids of neural networks and their applications. By the end of this article, you will have attained an extensive understanding of what neural network is.
+
+### Neural Networks Overview
+
+Neural Network is an operational element of deep learning. Deep learning uses neural networks, which mimics human behavior to solve problems. The neural network takes in data. Neurons recognize the patterns and predict the output.
+
+A neural network is a technology that uses network functions to understand and convert input data into the [desired output](https://deepai.org/machine-learning-glossary-and-terms/neural-network). The neural network's inspiration is by how the human brain works.
+
+The essential building block of a neural network is a perception. Perception accomplishes simple signal processing. Neural networks are pertained to solve data problems in Speech recognition, image recognition, and pattern recognition, to name a few.
+
+### How Neural Networks work
+
+Neural Networks are complex systems with artificial neurons. The neurons receive many inputs and process a [single output](https://www.upgrad.com/blog/neural-network-architecture-components-algorithms/#:~:text=Neural%20Networks%20are%20complex%20structures,input%20into%20a%20meaningful%20output).
+
+Neural networks comprise layers of neurons. The layers consist of the:
+
+- [Input layer](https://stackoverflow.com/questions/32514502/neural-networks-what-does-the-input-layer-consist-of)
+- [Multiple hidden layers](https://deepai.org/machine-learning-glossary-and-terms/hidden-layer-machine-learning#:~:text=In%20neural%20networks%2C%20a%20hidden,inputs%20entered%20into%20the%20network.)
+- [Output layer](https://www.scan2cad.com/user-manual/the-output-layer/)
+
+The input layer receives data represented by a numeric value. Hidden layers perform most computations required by the network. Finally, the output layer predicts the output.
+
+In a neural network, neurons dominate one another. A neuron is an object that holds a numeral value that is between 0 and 1. Each layer is made of neurons. Once the input layer receives data, it is redirected to the hidden layer. At the hidden layer, computation occurs, and information is sent to the neurons of the output layer. Neurons in different layers are connected through channels. Each channel is assigned a number referred to as weight.
```

Comment:
Factual Error: A neuron need not be limited to 0 & 1. It is the activation function that limits the output of the neurons.